src/components/MyDashboards/MyDashboards.tsx

```diff
@@ -0,0 +1,58 @@
+import TextField from "@mui/material/TextField";
+import { ChangeEvent, useEffect, useState } from "react";
+import dashboardCommunicator from "../../communication/dashboardCommunicator";
+import { ReactComponent as Icon } from "../../icons/search.svg";
+import { Dashboard } from "../../types/entities";
+import DashbaordItem from './DashboardItem/DashboardItem';
+import './MyDashboards.scss';
+
+const MyDashboards = () => {
+    const [searchedValue, setSearchedValue] = useState("");
+    const [myDashboards, setMyDashboards] = useState<Dashboard[]>([]); // need recoil
+
+    useEffect(() => {
+        (async () => {
+            const dashabords = await dashboardCommunicator.getMyDashboards();
+            setMyDashboards(dashabords);
```

Comment:
you wrote dashboards in 3 different ways so far 🤠

Reply:
חחחחחחחחחחחחחחח